docker-compose.yaml

```diff
@@ -0,0 +1,130 @@
+version: "3.8"
+services:
+    ollama-server:
+        image: ollama/ollama:0.1.39
+        pull_policy: always
```

Comment:
why always?

Reply:
`if-not-present` works? I guess the version will not get updated anyways. will change this.